internal/replacer/replacer_test.go

```diff
@@ -14,6 +14,8 @@
 //
 // SPDX-License-Identifier: Apache-2.0
 
+//go:build integration
```

Comment:
Why are these not `unit` tests? It certainly was the intention.

Reply:
There is a dependency between `image_test.go` that uses and `replacer_test.go` that defines a constant `testDigest`. `image_test.go` seemed like an integration test (took more than 500ms), now that I revisited it this is not the case, so I'll remove the `integration` tag from this

Reply:
Timeout [exceeded](https://github.com/hacbs-contract/ec-cli/actions/runs/3195743393/jobs/5216779685) for the test in `replacer_test.go`, so I needed to reintroduce the `integration` tag.